apps/pygame_demo.py

```diff
@@ -172,7 +176,7 @@ def draw(self, surface) -> None:
 
         # display a bit of use info on the display
         f = pygame.font.Font(pygame.font.get_default_font(), 20)
-        i = f.render("press any key for next map or ESC to quit", 1, (180, 180, 0))
+        i = f.render("press any key for next map or ESC to quit", True, (180, 180, 0))
```

Comment:
same boolean doubt, mypy was flagging this, the 2nd position of render is antialias, I supposed the 1 is True, right?

Reply:
I saw the same reply on pyscroll